Mathlib/Tactic/TypeCheck.lean

```diff
@@ -0,0 +1,9 @@
+import Lean.Elab.Tactic.Basic
+
+syntax (name := typeCheck) "type_check " term : tactic
+
```

Comment:
The `elab` command below will create the syntax for you. (It's only with `elab_rules` that you need to specify the general syntax explicitly first.) Removing this will avoid the linting error in CI.
```suggestion
```

Reply:
Aha, I was wondering how to fix that linting error. Thanks for the info!